packages/playground/optimize-deps/index.html

```diff
@@ -70,4 +73,7 @@ <h2>Dep with changes from esbuild plugin</h2>
 
   import { hello } from 'dep-esbuild-plugin-transform'
   document.querySelector('.esbuild-plugin').textContent = hello()
+
+  import { greeting } from '.hidden-dir/foo.js'
```

Comment:
I am a bit worried about if we should advocate this usage (which to me this a bit ambiguous). Wouldn't it better to do this way? (or am I missed something?)

```js
  import { greeting } from './.hidden-dir/foo.js'
```

Reply:
Good point :thinking:

Reply:
i fixed this, can this be merged now?